library/src/main/java/com/prolificinteractive/materialcalendarview/AnimatorListener.java

```diff
@@ -0,0 +1,13 @@
+package com.prolificinteractive.materialcalendarview;
+
+import android.animation.Animator;
+
+public class AnimatorListener implements Animator.AnimatorListener {
```

Comment:
Can we make this package local (remove `public`)? It shouldn't be part of the public api.